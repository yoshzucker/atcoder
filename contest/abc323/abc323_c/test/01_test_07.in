97 91
700 1000 800 800 500 2100 800 800 2200 1200 2300 2500 2000 1400 500 1300 2500 1600 800 2100 1400 1700 700 2500 900 2200 600 1400 2300 2300 1000 800 1000 900 900 1900 500 1300 500 1400 500 1100 2400 1200 800 900 500 2000 800 1900 900 1800 1000 1100 500 1200 800 1000 1000 1400 700 2500 2500 1300 1800 800 1100 2500 1500 1600 1800 2300 700 1600 2000 800 2300 2100 1800 900 1400 500 2200 2400 2100 1800 2300 1700 1200 1700 500
ooxooxooxoooxooooxxoxooxxxoxooxooooxxxoxooxooxoxoooooooxoxxxoxxoxxoxxxoxooxooxxxxooxoxxxoxo
ooooxooxxxxxooooooooxxxxxxooxxxxxoxoxooooxoxxxooxxooxxoxxxxooxxxoxoxoxxooooooxoooooxooxxxxo
xxxoxxoxoxxxxxxoxxooxxoxxxoooxooxooxxooooxxxooxxxooxoooxxxxxoooooxooxxxxooooxoxxoxoxxooooxx
oxxxxxxxxxoxxoxoooxooooxooooxooxxoxooxxxxxoooooxxxxooooxxoxxxoxoxxooooxxoxxxxxxoooxooxxoxxo
ooooooxoxoxxoxxoxoxxooxxxxxoxoooxooxoxxxxoxoxxoooxxxxxxoxooxxxxoooxxxxooxxxxooxxxxooxxoxoxx
ooooooxooxoxooxooxooxoxoxxxoxxoxxxxxxoxxxooxxooxxxoooxoxxoooxoooooxoxoooxoxxooxoxoxxxoooxoo
xoxxoxxxxooxxoooxxoxoxxxxxxxxxxxxxoooxooxxxoooxoxoooxxxoxxoxooooxoxoxoxooxxoxoxxoxxooxoooxo
xxoooxxoxxxoxxoxoxoxoxoooxooooxxooxxxxxoxoooxxooooooooxxxoxoxxoooxoxxxxxoxxxxoooooooxxxoxxo
xoooxoxoxxxxoxooxooxxxooxxxooxoxxxoooxoxooxxoxoxxoxxooxoxooooxxxooxooxxoooooxxoooxooooxxxxx
ooxxxxoxxooxxxoooxxoooooxxooooxxxoooxooxoxoooooxoxxoxxoxxxxxxoooxoxxxxoooxooooooxxoxooxooxo
xooxoxoooooooooxooxxoxxooxoooooxooxxoxoxxxxxxxoxooxoxoxoxoxxxoooxxxoxxxooooxoxoxxoxxoxxxoxo
oxxxoooooxxxxxxoxoxxxooooxxxxooxxxxooxooooxoooxooxoxxoooooooooxooxxooxxooxxxooxooxoxxooxoxo
ooxxxxxxxoxxxoooxxxxoxxxxxxxoooooooooxxxxxxxxooooxoxxoooxxoxooxxxxoxxoxoxoxxxooooxxoxooxxox
oxxxxxoooxoxxxxxxooxooooxxxxoxooxxooooxoxooooooxxoxooxooooxoxxooxxxoxxooooxoxxoxxxxoxxxoxoo
ooooxoooooooxxoxoxxooxxxooooxxxxxxooooooxoxxooxxoxxoooxxooxoxxoxoxxxooxoxoooxoxxoxooxooxoxx
xoxxxxoooxxooxoxooxxxoxxxoooxxxoxoxxxoxooxxooxooxooxxoxoxoxoxooxxooxxoxxxoxxxxooxoxxxoxxxxx
xxxxooxxoooooxxxxoxoxxoooxxxxoxoxooxxoxoxxooooxxxxxooxoxxoxxxoxxooooxoxxxooxxxxxooxooxxoxox
oxxoxooxxxooxxxxoxxxxoooxxxoxxxoxxoooooooxooxxxooooxxxoooxxxxooooooooxooxxxxxxoxxxxooooxxox
ooxoxoxxooxoxxooxxoxooxxooooooxooxoxxxoxooxoxxxoxooxoooxoxxxooxoooooxxxxxoooxoxooxoooxxoxox
oxoxooxxxoxxooooxooxxooxooxoxxoxoooxxooxoxoooxoxxoxoooxxooxxooxooxoxxxxxoooxxoooxxxxxxxoxox
ooxxoxoxooooooxxxoxxxooxoxxxxxoxxxoxxoxxoxxooxxooooxooxooooxxooxooxooxoxoooxoxoooxxxxxxoxoo
xoooxoxooooxoxxoxooxxoxxooxxoxoooooxxxoxoxoxoxooxoooxxxooxxooxxxxxxooxxooxooxoxooxoxxxooxxo
xooxxoooxoxoooxooooooooxooooooooxoooxxoxxooxxxoxooooxooxxoxxxoxoxoxxxxoooxoxoooxxxxxoxxooxx
oxxoxxooxxxoooxooooxoxxxoxxxxoooooxxxxxooxxooooxoxxoxxxxooooxxooxxooooxxxxxxoxoxoxxoxxooxoo
xooxoxxxxoxoxoxoxoxooxxoxoxooxxoxxooxxoxoooxxxoxxxoxoxoxxoxxxxoxooxxxoxxoxxxoxoooxxooxoxxox
ooxoxoxxooooxxoxooxoxoxxxxoxooxxxxxxxooooxxxxoxoxoxoxxoooxoxooxoooxxoxxxoxoxoxoxooxoxooooxx
xoxooxxxxoxoxooxxoooxxoooxoxoxxxoxoxxooxoxxooooxooxxooxxoxxoxoxxxxoxoxxooxxxxxoxxxoooxoxoox
oxxxooxxooxxoxxxxxooxxxxxooxxxxxxooxoxxxoxxooooxxoxoxxoxoxoooxoxxxxxoxxxooxxxooxooooxxxoxxx
oxxxxoxooooooxxxxxxoxooxoxoxxoooooxxxxoxooxxoxooooxxoooooxoooxxxxoxxxxoxxoxxxoxxoxxxoooxxxx
oxxoooxxxxxoooooxoxooooooxooooooxoooooxxxoxxxoooooxxxoxooooooxxooxxoooxxoxxxxxxoxxoxxooxooo
ooooxoooxxoooxoooooxooxooooxoxxoxooxxooxoxxoooxxxooxooxooxoooxoxxxoxxoxoooxxxxxoxoxxoxoxoxo
oxxxxxxoxoooxoooooxoxooxxoxoxxoxxxoooxooxoxxoooxoooxxxooxoxooxoxoxooxxxxxoxoooxxxxxxoooxxxo
oxxoxooxoxxxoooxxxxxoooxoxxoxxxooxxxoxoxxoxoxxoooooxoxoooxxoooxoxoxxoxxxoxxoooxoxoxxooxooox
xxoooooxoxxoooooooxoxxxoxxoxoooxxoxooxoxxoxoooxoxxxoooxooxooxxoxxxoxxoxxxxxxooxoxoxoxxoxxoo
xxoxoooooooxoooxxxxxxooxxxoxoxxoxooxxoxxxoooooxxxxxoxxxxoxxxooxooxoooxooxooxxooxoooooxxoxoo
oxoooxxxoxxooxxoxooxoooxxooxoxxxxxxxoxoooxxxxooxooxxoxoxxoooooxoxxooxooxxxoxxxxooxooooxoxxo
oxoxooooxxooooxoxoxxxoooxxoxxoxxxxoxxooxxoxxoxxooxxooxxoxxxxooxxooxxoxxoxxoxxooxoxxooxoxoox
oxooxoxxoxooxxxooooxxooxooxxooxxoxooxxxxoooxoxoxxooxoxxoxoxoxxooxxoxxxoxoxxxxoooxxoxxxooxox
xooxoxoxooxoxxooxoxoxoxooooxoxxoxoooxxooxxoooxxooxxxooxxxxooxxoxoooxooxooooxxoxxxxxxxooxooo
xxxxoooooooxoxxxxoxoxxooooxoxoxxooooxooooxoxxoxooxoxxxoxxxxxoxoxooxoxxxxxxxooxxxxxxxxxooxxo
xoxoxxoooxxxxoxxoxxxxxxooxxxoooxxxxxxoxxoxxxxoxoxxoxxooxoxoxxxxxooooxxoooxxooxooooxxooooxxx
xxoooxoooxoxxxoxooxxoooxxxoxooxoxxxxooxxxxxoooxxxxxoxxooxxxxoooxxoxooxxoxxoooxoxoxxxxoxoxxx
xoxoxxoxxxxooooxxxoxxoxoxxxxxoxooooooxxxoxoxxxoxoxoxoxoxxooxoxxxoxooxxoxooxxxxxooxoxoxxxxox
xoooxxxxoxooxxxxxxxooxooxxxoxxxoxxoxooxxoooxxoxoxoxooxooxooxooxxxoxooooooooxxxooxoxxxxxxooo
xoxxxxoxxoxoxoooxoooooxooxxooxxxoooooxxxxooxxxxoooxoxxooxxxoxoooxxxoooxoxxxooxxxoxoxxooooox
xoxxxoxooooxoxooxxxooooxxoxxoxoxooxoxooxxxxxxxxooxxxoooxoxxxooxxxoxxxoxoxxoxooooxoooxoxoxoo
xoxxoooxooxoxoxoxooooooxoxxxoxxxooxxoxoxxxooxoxxxxoooxxxoooxoooxooooooxxxxooxoxoxxxxoxoxxxx
oxoxoxxxoxxoxoxxxxxoxoxxooooooxooxooooxooxxxxoxooxxxooooooxxoooooxxoooxxooooxoxoooxooooxoxx
oooxoooxoxxoxoxoooxoxoxoxoooxoxooxooxooooxooxoxooxxxxooooxooooooxxoxoxxxooxxoooxooxoxoxxxxo
ooxxoxoxxooxxxxoxoooxoxxxxxxxoooxooxxoooxoxxoxooxxxxoxxxxxxxxooooxxoxxoxoooooxxooxxoxoxxoxx
xoxoxxxoooxxooxooooooxxxoxxxxxxoxoxxxooooxoxoxxooxoxoooxooooxxxoooxxxoxoxoxxxxoxxxxoxxoxoxo
ooooxxooxooxoxxxooxxoxooooxoxoooxoooxoooxooooxoooooxooxxoxooxxxooxooxxooxxooxxxoxxooooxxoox
oxxxoooxxoxoooxxooxoxxxoooooxoooooxxxxoxxxxooxxoxxoooxxooxxooxoooooooxxoooxxxxooxoxxooooxxx
oxxoooxooxxoxxxxxoxooooooxxooxoxxxxoxxoxxooxxoxoxoooxxxxoooxooxoxxoxoxooxooooxoxxoxxooooxox
ooxxxoxoooxxxxoxxooxoxxoxxxoxxxxxxxoxxxxoxxoooooooxxxxoooxoxoxxoxxooxxooooxoxoooxxooxxxxxox
xoooxxoxoxoooxooxoxxxxxooxxxxoooxxoxooxoxoooxoxxxxooxooxxxooxxoooxxxxxxxxooxxxxxoxoxxoxooxo
oooxooxxxoxoxxoxxxoxoxoxooooooooooxxoxoxxxxxxxoxoxoxoxxoooxxxoxoxoooooooxoxoxxooxxoxxoxxoxx
oxoooxooxxoxxooxxxooxxxoxoxooooxxooxoxoxxoxoooxoooxoxoxxxooooooxoxxxoxoooxxoxooxxxooooxxxxx
oooxxoxxooxooooxxxoxooxxxoxooooooxoxxoxxxxooxoooooooxoxoxooxxoxxxooxxoxoxxxxxxooxxoxxxoxxxo
oxoooooooxoxooxxxoxoxoxoooxxoxxoxoxxoooxooxoxoxooxooooooxxxooxooxoxoxoooxoooxoxoxxxxoooxxxo
oxxxooooxoxxxoxxoxxxxxoxxxooxoooxxxxxxxoxxoxooooxoxxooxoxxooxxooxxxoxxxoooxooxooxxoxoxxxoxo
xxxoxxoooxooxxxoxoxxxooxoxoooooooxoxxooooooxxoooxoxoooxxoxxoxxoooxoxooxoxooxxxxxoooxooxooxx
xxoxxoxxooxooxxxooxxoxxooxxooooxoxooxxxoxooooooxooxooooxxxoxoooooxxooooooxoxxxoooxooxxxooxx
xxooooxxxxxoxoxxxoxxxxoooxxooxoooxooxooxxxooooxxxooxoxxxxxxoxxooooooxoxoxooooxxxoxoxxooxxxo
oooxxxxxoxxooxoxoxoxooxoxoxxxxxxoxooooxxooxxxooooxoooooxxxxoooxxooooxooxooxxoxxooooxooxxxoo
xooxxxoxxxooxxoxxooxxoxoxooxoxoxxoxxxooxxxooxxoxoxxxxxxoooxoxxooooxooxoxxooxxooxxxoxxxoxoxo
xxooxxoxooooxxxxxxxxxxxxxxxxoxoxxoxoxxoooxoxxoxxoxxoxxxooxxxxoxooooxxoooxoooxxoxxooooxoxxxx
ooooxooxxxoooxooxxxxoxoxoooxxxxxooxooxxxxxxxooxxoxxxooooxooxxxxxoxooxoxooooxoxoxooooxooxoxx
xxooxooxxxooxoxooooxoooxooooxxxooxxxooooooxxxxooxoxoxoxooxxoxxooxoooooxxoooxooxoxoooxooxoox
ooooxoxoxooxoooooooxooxxooxoxoxooxooxxoxxooxxoxxooxxxxxooooxxooxxxoxxxxxooxoxxxxooxxxxxooox
oxoxxxxxxoxxxxoxooxooxxxxxoxooxoxxoxoxxxxoxxoxoxxooxxxoxoxoxooxoooxooxoxoxxxoooxooxxxooxxxx
oooooxoxxoxoxoooxxoxxxxxooxoxxxoxxxxoooxxxxxxxoxoooxxoxoxxxxxxxoxxxxoxxooxxoxoxoxooxxxxoooo
oxoxxoxooxxxoooxoxxxxoxoxxoxoxxoxoxooooxooooxoxoxxoxxxooxoxxxxoooxooxxoxoxoxooxoooxooxxoxox
xxxxxooxoooxoxooxxoooxoooxoooxoxxoxoxxxxxxoxxxoxooooooxxoxxoxxxxxxxoooxxxoxxoooxoxoooooxoox
oxoxxoxoxxooxoxooxxoxoooooooooooxxoooooxoxxxoxxoxxxoxxxxxoxxxxoooooooxxxxoxxoooxoxooxxxxoox
oxoxoooxooooxooxoxxoxxxooxooxxxoxoxxxoxxxxoxoxoooooooxxooxxxooxoooooxoooxxxxxoxoooxxxoxxoxo
xooxooxoxoxooxooxoxxxxxoxxxxxxoooxoxxoooxxxxooxoxoxoxxxoxoxxoooooooxxxoooxoxxoxxxxoxxoxxoxo
oxxxxoxoxoxxoooxxxxoxoxxxoxooooooxooooooxxooxooxxxxooxxooxoooxxoxoooooxoxxoooxooxoxoxxoxoox
xxxoooooxoooooooxoxxxxooxxxoxoxooxxxoooooooxooxxxooxoxxoxooxxxxxoooooxxxxooxxooxoxxoxxxooxx
ooxooxxoooxxxoxxoooxxxooxxoxxxoooooxoxxoxoxooxoxxxooxxxoxxooxoxoxxxxoxoooxoxxoxxxooxxxxooxo
ooxxxoxxoooxoxxoxxxoooooooxoooxxoooxxxxooooxooxoooxooooooxxooooxxoxoxooooxxooooooooxxooxxxo
ooxxoxxooooooxxooooooooooxooxxxoxoxxxoooxoooxxxooxxxxxxooxoxoxooxoooxoxoxxoxxooxoxxooxxxxoo
oxoxoooxoooooxooxoxoooooooooxxxoxxoooxxoxoxooooooxoxxoxoxxxxxxoooooooxoooxxxxoxooxxoooxoxxx
xxoxxoxoxooxoooxoxoxooooooooxooooxoxxoxoxooooxooxxoxxxoxxoxxxxxooxxoxooxxxoooxooooooxxooxox
xxxxooxoxoooooxxoxxoxxooooxxoxoxooxoxxxoxoxxxoxxooooxoxooxoxoxxoooxoxxoxoxoooooooooxxxoooxo
ooxooxxxooooxooxoxooxxooxoxoxooxxoxxxoxoxxxxxxxoxooxxoxoxooxxxoxxxxoxoxoxxxooooooxoxoxoooox
xoooxoxxoxxooxoooooxooxoxxoxxoxoxxxxooxoooooxxooxxxooxoooooooxxxoxoxoooxoxxoxxoooooxoxoxoxx
oxoooooxooxooooooooxxxxoxxooxoxoooooxxxoxoxxxxxxoxxxxooxooxxoxxooxxooooxxoooxoxoooxooxxxxxx
xxxxxooxxoxxoooxxoxoxoxoxxooxxooxxoxxxxoxxoooxoxxxoxxoooxxxxxooxxoxoxoxxxoxxooooxxxooxxxoox
xxooxxxoxoxoxoooooxxooooxoxoooooxxoooooxooooxoxxoxxxoxoxooooooooxxoooxxoxxooxxxxxxxxxxooxxx
xxxxoxxoxxxxxoooxxxoxxoxxxxxooxxoxxxxoooxoxxoxooxoxxxxxxxoxxooxooxooxxooxoooooxooooxxoxxxxx
ooxoxxoxxxooooxxoxxxxxooooooxxxoxxoxxxoxxxxxxoooxoxxxxxoxoooxxoooxoxoxoooxxxxxooxxxoxoxxxox
oooooxoxxoxooxxooxoxxxxxooxoxoxxxoooxoxxxxooooxxxooxxoxoxooxooxoxxooxxxoooxxxxxxooxoxooooxx
oxxooxooxoooxooxoxxooxxxxooxxxoxoooxoxoooxooxoxxooxooooxxooxoxoxoxoxxxxoxxooxooooxxxoxxoooo
ooxxxxoxoxoxooooxxooxxxxoxoxoxoxxxoxxoxooxxxooooxxoooooooxoxxxooxxxxxooooxooxoxxoooxxxoxxox
oooxooxxoxooxxxxoxooxxoooxxoooxoxooooooxoxoxxxoxooooxxoxxooxxxxoxxxoxxoxxoooxoxxoxxoooxxxxo
xxooxoxoxooxxooxxxxxxoxooooxooxooxxoxxxoxxooooxxoxooxxoxoxooxxoxxxooxxoxoxooxxoxxxxoxoxoxox
