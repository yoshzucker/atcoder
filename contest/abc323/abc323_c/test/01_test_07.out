11
10
12
12
14
5
13
10
10
6
8
8
15
10
6
15
11
8
7
12
9
9
4
11
13
8
13
17
13
6
8
10
13
9
7
11
10
9
7
13
14
14
16
11
6
9
10
5
4
13
12
3
7
6
14
12
12
8
9
5
14
6
3
9
8
12
13
10
4
10
15
18
8
7
6
8
12
7
9
14
0
6
4
5
5
7
7
9
13
8
14
13
10
9
11
9
11
